### PR TITLE
Putting `composer update` on a diet

### PR DESCRIPTION
**Pull request by bobdenotter**
This PR "unpacks the packs", tweaks the version constraints and sets `stability stable`. 

Before: 

 - time: 1:23,52 total
 - memusg: peak=1660 Mb

After:

 - time: 1:04,45 total
 - memusg: peak=851 Mb

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -6,34 +6,47 @@
...
+        "nelmio/cors-bundle": "^1.5",
         "nesbot/carbon": "^1.34",
         "php-translation/symfony-bundle": "^0.8.1",
+        "phpdocumentor/reflection-docblock": "^3.0 || ^4.0",
         "sensio/framework-extra-bundle": "^5.1",
         "sensiolabs/security-checker": "^5.0",
         "siriusphp/upload": "^2.1",
         "stof/doctrine-extensions-bundle": "^1.3",
-        "symfony/asset": "^4.2",
+        "symfony/asset": "^3.0 || ^4.0",
```
Comment (JarJak) [inline composer.json]: should be ^4.2

File: composer.json [hunk 2]
```diff
@@ -6,34 +6,47 @@
...
         "symfony/console": "^4.2",
-        "symfony/debug-pack": "^1.0",
+        "symfony/debug-bundle": "^4.2",
+        "symfony/expression-language": "^3.0 || ^4.0",
```
Comment (JarJak) [inline composer.json]: should be ^4.2

File: composer.json [hunk 3]
```diff
@@ -6,34 +6,47 @@
...
         "symfony/flex": "^1.1",
         "symfony/form": "^4.2",
         "symfony/framework-bundle": "^4.2",
-        "symfony/orm-pack": "^1.0",
+        "symfony/monolog-bundle": "^3.0",
         "symfony/polyfill-php72": "^1.8",
+        "symfony/security-bundle": "^3.0 || ^4.0",
```
Comment (JarJak) [inline composer.json]: should be ^4.2

File: composer.json [hunk 4]
```diff
@@ -6,34 +6,47 @@
...
         "symfony/serializer": "^4.2",
+        "symfony/stopwatch": "^4.2",
         "symfony/swiftmailer-bundle": "^3.1",
         "symfony/translation": "^4.2",
+        "symfony/twig-bundle": "^4.2",
+        "symfony/validator": "^3.0 || ^4.0",
```
Comment (JarJak) [inline composer.json]: should be ^4.2

**Comment by bobdenotter** [comment]
Now: 

 - time: 1:01,81 total
 - memusg: peak=786 mb

**Pull request merged**